docs/examples.rst

```diff
@@ -53,29 +48,32 @@ Example: synthesizing 1000 soundscapes in one go
     time_stretch_min = 0.8
     time_stretch_max = 1.2
 
+    # generate a random seed for this Scaper object
+    seed = np.random.randint(0, 100000)
+
+    ## alternate ways to define random state:
+    #     seed = np.random.RandomState(0)
+    #
+    # or don't define any random state. runs will be random and not reproducible in 
+    # this case. you can use np.random.get_state() to reproduce the run after the fact
+    # if needed:
+    #     seed = None
```

Comment:
This feels like it belongs more in a tutorial (i.e. in the docs) than in this example. Let's remove it from here.